setup.py

```diff
@@ -230,11 +234,11 @@ def get_extension_modules():
     packages=["turbodbc"],
     setup_requires=[
         "pybind11>=2.2.0",
-        "pyarrow>=1,<9.1.0",
+        "pyarrow>=1,<10.0.2",
```

Comment:
Is there any reason to expect that things would break in a patch release?

Why not `<10.1` or even `<11`?

Reply:
I'll change it to 11, that's what it's in the test